packages/spark-core/components/_alerts.scss

```diff
@@ -0,0 +1,64 @@
+// ==================================================================
+// Alerts
+// ==================================================================
+.sprk-c-Alert {
+  background-color: $gray-light;
+  border-radius: $corners;
+}
+
+.sprk-c-Alert,
+.sprk-c-Alert__icon {
+  display: flex;
+}
+
+.sprk-c-Alert__icon,
+.sprk-c-Alert__text {
+  padding: $space-inset-m;
+}
+
+.sprk-c-Alert__text {
+  align-self: center;
+  flex: 1;
+}
+
+.sprk-c-Alert__icon {
+  align-items: center;
+  justify-content: center;
+  width: 4.375rem;
+}
+
+.sprk-c-Alert__icon:first-child {
+  border-top-left-radius: $corners;
```

Comment:
I added a $corners var to the main settings file because we should have consistency across the system (in most cases) when it comes to the feel of ui pieces.